backends/events/sdl/sdl-events.cpp

```diff
@@ -270,8 +270,14 @@ bool SdlEventSource::handleKbdMouse(Common::Event &event) {
 			}
 
 			if (_km.x != oldKmX || _km.y != oldKmY) {
+				// keep hi-res coordinates since 
+				// processMouseEvent will overwrite them with lo-res numbers
+				oldKmX = _km.x;
+				oldKmY = _km.y;
 				event.type = Common::EVENT_MOUSEMOVE;
 				processMouseEvent(event, _km.x / MULTIPLIER, _km.y / MULTIPLIER);
+				_km.x = oldKmX;
```

Comment:
I have made another change after I realized that very slow analog pointer motion was still capped at 1 pixel per delay interval.

This was because processMouseEvent overwrites the fine-grained hi-res _km.x coords with coarse-grained lo-res ones (basically modulo MULTIPLIER). A simple variable safe and restore fixes this.